cmake/SetupPackages.cmake

```diff
@@ -82,6 +82,14 @@ endif()
 
 
 if (RAJA_ENABLE_TESTS)
+
+#
+# This conditional prevents build problems resulting from BLT and
+# RAJA each having their own copy of googletest.
+#
+if (RAJA_BUILD_WITH_BLT)
```

Comment:
If it's turned on in standalone RAJA, the tests won't build. So I don't think it should appear in any RAJA CMakeLists.txt file.  I view this as a short-term band-aid so that the perf suite works with the develop branch of RAJA until we decide to switch over to BLT.